src/main/huggingface.py

```diff
@@ -126,6 +130,63 @@ def predict():
                     mimetype=mimetype)
 
 
+@huggingface_api.route('/download-model', methods=['POST'])
+def download_model():
+    """
+    download specified models.
+    @return: Return if the status of downloading models.
+    """
+    body = request.get_json()
+
+    if not body:
+        return error_message(missing_body_error_msg, 400)
+
+    paths = {
+        modelST.field_name.lower(): modelST.path,
```

Comment:
Same here... Change `modelST` to `modelSentenceTransformers`. Same for `modelQA`. For now we have two, but in the future there could be more uses cases, such as Text Summarizing. It can get confusing at that point.